scripts/vmss/vmss-status.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+# set -x
+shopt -s nocasematch
+
+# Source shared function scripts
+source scripts/vm/common-functions.sh
```

Comment:
Should it be VMSS` common-functions.sh` ?

Reply:
made the change for this - thanks